db/seeds.d/16-mail_notifications.rb

```diff
@@ -49,11 +50,19 @@
     :description        => N_('A test message to check the email configuration is working'),
     :mailer             => 'UserMailer',
     :method             => 'tester',
-    :subscription_type  => false
+    :subscriptable      => false,
+    :subscription_type  => nil
   }
 ]
 
 notifications.each do |notification|
-  MailNotification.create(notification) if MailNotification.where(:name => notification[:name]).blank?
+  notification[:name] = notification[:name].to_s
```

Comment:
I can't see one, especially as it's simply stored via AR, it'll always be a string.